Cargo.toml

```
@@ -27,10 +27,14 @@ incremental=false
 
 [dependencies]
 "alloc-no-stdlib" = {version="2.0"}
-"brotli-decompressor" = {version="~2.3", default-features=false}
-"alloc-stdlib" = {version="~0.2", optional=true}
-"packed_simd_2" = {version="0.3", optional=true}
-"sha2" = {version="~0.10", optional=true}
+"brotli-decompressor" = {version="2.5", default-features=false}
+"alloc-stdlib" = {version="0.2", optional=true}
+"sha2" = {version="0.10", optional=true}
+
+[dependencies.packed_simd]
+git = "https://github.com/programingjd/packed_simd.git"
+default-features = false
+features = ["into_bits"]
```

Can we put these inline inside the `[dependencies]` section?  Makes it more readable IMO, vs looking at a dedicated section.  Also, this will need to be tied to a specific git revision (or at least a tag or a branch).

```suggestion
"packed_simd" = { default-features = false, features = ["into_bits"], git = "https://github.com/programingjd/packed_simd.git" }
```